series-tmdb.en.yaml

```diff
@@ -611,6 +611,19 @@ entries:
       - season: 1
         anilist-id: 223
 
+  - title: "Dragon Ball GT"
+    guid: plex://show/5d9c080a02391c001f57f8f9
+    # imdb: https://www.imdb.com/title/tt0139774/
+    # tmdb: https://www.themoviedb.org/tv/12697
+    # tvdb: https://www.thetvdb.com/dereferrer/series/79275
+    seasons:
+      - season: 1
+        anilist-id: 225
+      - season: 2
```

Comment:
GT only has 1 season on TMDB

Reply:
I've removed the additional season entries